photopicker/build.gradle

```diff
@@ -43,7 +48,7 @@ dependencies {
     // misc
     implementation 'com.squareup.picasso:picasso:2.71828'
```

Comment:
:(, so we have Glide, Coil, Picasso for loading images.

Reply:
Sadly, but we can update Unsplash to use Coil in the future to remove the Picasso depency. My guess is that is should be straightforward.